helpers/context.go

```diff
@@ -144,10 +163,15 @@ func (context *ConfiguredContext) Teardown() {
 			runner.NewCmdRunner(cf.Cf("delete-user", "-f", context.regularUserUsername), context.shortTimeout).Run()
 		}
 
-		if !context.isPersistent {
+		if !context.isPersistentApp && !context.isPersistentOrgAndSpace {
 			runner.NewCmdRunner(cf.Cf("delete-org", "-f", context.organizationName), context.shortTimeout).Run()
 			runner.NewCmdRunner(cf.Cf("delete-quota", "-f", context.quotaDefinitionName), context.shortTimeout).Run()
 		}
+
+		if context.isPersistentOrgAndSpace {
+			runner.NewCmdRunner(cf.Cf("t", "-o", context.organizationName), context.shortTimeout).Run()
+			runner.NewCmdRunner(cf.Cf("delete-space", "-f", context.spaceName), context.shortTimeout).Run()
+		}
```

Comment:
@coreygibbs I'm not sure I understand why you are deleting the space if it is meant to be persistent.  I might be missing something here, could you help explain?